src/static/css/content.less

```diff
@@ -316,6 +322,19 @@ figure {
 // Initiate loop
 .color-swatch-loop( length(@colors) );
 
+.color-table {
+    width: 100%;
+    margin-bottom: 40px;
+
+    .swatch_field {
+        width: 200px;
+        height: 2.5em;
```

Comment:
Not a blocker to release if we need to have a discussion about it, but I'd like to add `border-bottom: 1px solid @white;` here, to override the row's standard gray border between color swatches.